test/EndToEnd/GivenWeWantToRequireWindowsForDesktopApps.cs

```diff
@@ -48,7 +49,24 @@ public void It_does_not_download_desktop_runtime_packs_on_unix()
             testProjectCreator.AdditionalProperties["OutputType"] = "exe";
             testProjectCreator.AdditionalProperties["RuntimeIdentifier"] = Rid;
 
-            var testInstance = testProjectCreator.Create();
+            // At certain point of the release cycle LatestRuntimeFrameworkVersion in eng folder may not exist on the nuget feed
+            static void overrideLastRuntimeFrameworkVersionToExistingOlderVersion(XDocument project)
+            {
+                XNamespace ns = project.Root.Name.Namespace;
+                var target = XElement.Parse(@"  <ItemGroup>
+    <KnownFrameworkReference Update=""@(KnownFrameworkReference)"">
+      <LatestRuntimeFrameworkVersion Condition=""'%(TargetFramework)' == 'netcoreapp3.1'"">3.1.10</LatestRuntimeFrameworkVersion>
```

Comment:
Are we going to want to keep this updated to N-1 and is there a good way to do that?